core/kazoo_json/test/kz_json_test.erl

```diff
@@ -55,18 +56,15 @@ prop_get_value() ->
                       end)
            ).
 
--define(DO_NOT_RUN_QC_SET_VALUE, true).
--ifndef(DO_NOT_RUN_QC_SET_VALUE).
```

Comment:
Could you fix this one too? https://travis-ci.org/2600hz/kazoo/jobs/171549029#L2398